app/models/session.rb

```diff
@@ -208,10 +208,6 @@ def as_json(opts=nil)
       end
     end
 
-    if type == "MobileSession"
```

Comment:
This would prolly still used. I'll double check with Radek if it creates problems with the mobile app